Content.Server/Ghost/Roles/GhostRoleSystem.cs

```diff
@@ -409,6 +415,59 @@ private void OnTakeoverTakeRole(EntityUid uid, GhostTakeoverAvailableComponent c
 
             args.TookRole = true;
         }
+
+        private void OnVerb(EntityUid uid, GhostRoleMobSpawnerComponent component, GetVerbsEvent<Verb> args)
+        {
+            var prototypes = component.SelectablePrototypes;
+            if (prototypes.Count > 1)
+                return;
```

Comment:
Am I missing something? Doesn't this stop it from working?

Reply:
If it has less than 1 selectable prototypes than it reverts to normal behavior (no verb)

Reply:
Wouldn't that be if (prototypes.Count < 1)?